src/core/file_sys/fs_save_data_types.h

```diff
@@ -41,6 +41,8 @@ enum class SaveDataRank : u8 {
     Secondary = 1,
 };
 
+enum class SaveDataFormatType : u8 { Normal = 0, NoJournal = 1 };
```

Comment:
```suggestion
enum class SaveDataFormatType : u8 {
    Normal = 0,
    NoJournal = 1,
};
```
(the trailing comma is needed to keep the linter from forcing it to one line)